doc/connectivity/bluetooth/bluetooth-contrib.rst

```diff
@@ -0,0 +1,55 @@
+.. _bluetooth-contrib:
+
+Contributing to the Bluetooth subsystem
+#######################################
+
+Overview
+********
+
+This page describes how one would contribute to the Zephyr Bluetooth stack.
+It is a work in progress.
+
+The first step is familiarizing yourself with the :ref:`Stack architecture <bluetooth-arch>`.
+
+.. _bluetooth-coding-style:
+
+Coding guidelines
+*****************
+
+The following describes a few of the coding conventions we use in the Bluetooth
+subsystem. See also the :ref:`Zephyr Coding Guidelines <coding_guidelines>` and
+the :ref:`Zephyr Coding Style <coding_style>`.
+
+The guidelines in this document should not take precedence on rules from the
+Zephyr Coding Guidelines and Coding Style mentioned above.
+
+Unless there is a convention listed here that disagrees, the general "rule" is
+to try to follow the style of the code surrounding the change.
+
+The Bluetooth contribution guidelines are not set in stone, and this is a living
+document. It will be updated if enough people agree on amending or changing the
+rules.
+
+Tools
+=====
+
+Install and use editorconfig_. This should ensure your editor uses the proper
+settings, e.g. indentation, maximum line length, etc..
```

Comment:
This tip isn't bluetooth specific.  Please move to the general zephyr guidelines.

Reply:
@keith-zephyr the reason that this is kept in Bluetooth is two-fold:

- All of these rules and exceptions are already being used and enforced in the Bluetooth subsystem
- The chance of this text ever being accepted in the general project in a reasonable timeline is slim. We have tried to make changes to the coding style before and we have never succeeded

In my eyes, I'd rather have what we already do documented than nothing at all.

Reply:
Why wouldn't we enhance: https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style